### ISSUE TITLE
Is the learning rate the same as mentioned in paper?

### ISSUE THREAD

**Issue by TianChenone** [tegg89/SRCNN-Tensorflow#35]
I found that the learning rate mentioned in paper is 1e-4 for first two layers and 1e-5 for the last layer and they find that a smaller learning rate in the last layer is important for the network to converge. But I didn't see the change of rated between different layers in this code. Waiting for your reply, thanks!
![image](https://user-images.githubusercontent.com/39474398/44498385-bbcc8d80-a6b0-11e8-89dd-574c77fd8756.png)

**Comment by TianChenone** [comment]
And the loss during my train decrease quite slowly and really not obvious.
After 914 epoch
![image](https://user-images.githubusercontent.com/39474398/44498463-2b427d00-a6b1-11e8-9b4d-9ea76e7c598e.png)

**Comment by JeasonUESTC** [comment]
Hi, how did you curve plotting ,could you share the way and code to me? Thank you very much.

**Comment by TianChenone** [comment]
Using tensorboard. Sorry, because I have given up the method, the code has been cleaned. Maybe you can google or baidu the 'tensorboard', it's quite easy to handle.
